### ISSUE TITLE
No sound?

### ISSUE THREAD

**Issue by m2pt5** [EBLeifEricson/orchestrina#8]
I don't know if I'm doing something wrong, but I'm getting no sound at all out of Orchestrina. (CIA or 3DSX version.) I am sure the songs are in the right place, but it's not even making the instrument noises. (Also it won't download the songs automatically, but I think that's because Github changed something a while back.) I'm using Luma 10.2.1 with Boot9Strap on an old 2DS, firmware version 11.14.0-46U. (Everything else plays sounds fine, just not Orchestrina.)

**Comment by GlimmerDev** [comment]
Hi, did you use something like [DSP1](https://github.com/zoogie/DSP1) first to dump the DSP firmware? Sound will not work without it.

This is an old application though at this point so it's entirely possible something is broken.

**Comment by m2pt5** [comment]
Ahh ok, I'll do that. It's been a long time since I used anything that needed that, and this is a newer system.

**Comment by GlimmerDev** [comment]
Yeah sorry about that, if I have time to revisit maybe I can do a new build and bring it more up to date. I'll try to fix the song downloads as well!
